### ISSUE TITLE
Spring Boot 2.x Actuator Endpoints

### ISSUE THREAD

**Issue by mgardiner** [stoicflame/enunciate#854]
Update Enunciate to support documenting Spring Boot's 2.x Actuator endpoints.

**Comment by panchenko** [comment]
IMHO that's already documented: https://docs.spring.io/spring-boot/docs/2.0.x/actuator-api/html/
@mgardiner what would like to achieve exactly?

**Comment by mgardiner** [comment]
@panchenko, the idea is to have Enunciate "automatically" generate HTML documentation for the Spring Boot actuators that have been enabled within a Spring Boot application to provide visibility into which Spring Boot actuators have been made public.

For example, lets say I wish to only enable the /actuator/info and /actuator/healthcheck Spring Boot actuator endpoints for my application; I would expect Enunciate to detect that during build time and generate the appropriate API documentation along with all my other REST APIs that I have written.

**Comment by panchenko** [comment]
The enunciate is indeed executed during build time, but you can change endpoint configuration after that. There is some mismatch here.

**Comment by mgardiner** [comment]
I agree there is some mismatch in that it is possible to override which actuator endpoints are enabled or not but to do so requires access to the server / Docker instance which in our case we don't allow in our production instances.
